backend-tests/run

```diff
@@ -5,73 +5,91 @@ HERE=$(dirname $(readlink -f "$0"))
 INTEGRATION_PATH=$(dirname "$HERE")
 export INTEGRATION_PATH
 
-COMPOSE_CMD="docker-compose -p backend-tests \
-             -f $INTEGRATION_PATH/docker-compose.yml \
-             -f $INTEGRATION_PATH/docker-compose.demo.yml \
-             -f $INTEGRATION_PATH/backend-tests/docker/docker-compose.backend-tests.yml"
-
-# by default just add minio, with COMPOSE_CMD_BASE this creates the standard onprem ST setup
-COMPOSE_FILES_DEFAULT=( "$INTEGRATION_PATH/docker-compose.storage.minio.yml" )
-COMPOSE_FILES=()
+COMPOSE_CMD_OPEN="docker-compose -p backend-tests \
+        -f $INTEGRATION_PATH/docker-compose.yml \
+        -f $INTEGRATION_PATH/docker-compose.demo.yml \
+        -f $INTEGRATION_PATH/backend-tests/docker/docker-compose.backend-tests.yml \
+        -f $INTEGRATION_PATH/docker-compose.storage.minio.yml"
+COMPOSE_CMD_ENTERPRISE="${COMPOSE_CMD_OPEN} -f $INTEGRATION_PATH/docker-compose.enterprise.yml \
+        -f $INTEGRATION_PATH/extra/recaptcha-testing/tenantadm-test-recaptcha-conf.yml \
+        -f $INTEGRATION_PATH/extra/smtp-testing/conductor-workers-smtp-test.yml \
```

Comment:
No idea how this file is used, but did we not just remove conductor?

Reply:
Good question ;)

So basically at the time of removing conductor, we wanted to rename this file (i.e. workflows-workers...) but then we had no mechanism in mender-qa to make in backwards compatible. The only solution was an ugly if this file exist...

That was the original motivation for this task. At the moment we cannot run 2.3.x backend-tests so that has ben the "second" motivation to do it.

Reply:
Ah, but then you can rename it now, right?

Reply:
I guess so.

But let me first merge this one (and do similar fixes in 2.3.x and 2.2.x); then separately I will ping Fabio to see if in master the file can completely be removed or only renamed.

Reply:
@lluiscampos @kacf I confirm we can rename the file. I couldn't rename it because, as Lluis pointed out, it would have broken the tests for 2.3.x.

Reply:
:+1: Feel free to rename it in master next time you are around that.

 Funny enough, the tests were broken for 2.3.x already when we introduced the `stripe-test.docker-compose.yml`. This will be fixed in #850